contract-ui/tabs/claim-conditions/components/claim-conditions.tsx

```diff
@@ -199,12 +196,10 @@ const ClaimConditionsForm: React.FC<ClaimConditionsProps> = ({
 }) => {
   const trackEvent = useTrack();
   const [resetFlag, setResetFlag] = useState(false);
-  const isAdmin = useIsAdmin(contract as ValidContractInstance);
+  const isAdmin = useIsAdmin(contract as unknown as ValidContractInstance);
 
-  // We're setting it as Erc1155 so TypeScript doesn't complain that we don't have a tokenId.
-  const query = useClaimConditions(contract as Erc1155, tokenId);
-  const mutation = useSetClaimConditions(contract as Erc1155, tokenId);
-  /*   const decimals = useDecimals(contract); */
+  const query = useClaimConditions(contract, tokenId);
+  const mutation = useSetClaimConditions(contract, tokenId);
   const decimals = 0;
```

Comment:
decimals for erc20 should be 18 (or whatever they have)

Add useTokenDecimals in react sdk